packages/react-components/src/components/ImageGallery.test.tsx

```diff
@@ -1,11 +1,15 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
-
+/* @conditional-compile-remove(image-gallery) */
 import React from 'react';
+/* @conditional-compile-remove(image-gallery) */
 import { ImageGallery, ImageGalleryImageProps } from './ImageGallery';
+/* @conditional-compile-remove(image-gallery) */
 import { render, screen } from '@testing-library/react';
+/* @conditional-compile-remove(image-gallery) */
 import { registerIcons } from '@fluentui/react';
 
+/* @conditional-compile-remove(image-gallery) */
```

Comment:
We don't need these 5 CCs because ImageGallery.tsx is not under CC so except the last two tests, all the other tests will pass

Reply:
Good call out. Removed.